schemas/com.snowplowanalytics.snowplow/media_player_session/jsonschema/1-0-0

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media player events that tracks a session of a single media player usage",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_session",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "sessionId": {
+      "type": "string",
+      "format": "uuid",
+      "description": "An identifier for the session."
+    },
+    "startedAt": {
+      "type": [
+        "null",
+        "string"
+      ],
+      "description": "Date-time timestamp of when the session started.",
+      "format": "date-time"
```

Comment:
Should this not be a timestamp type?

Reply:
I think a date-time is more user friendly. This info is the actual time of the start of the session, it doesn't relate to the playback time so I think it's better to represent it in a readable format.